src/viewer.js

```diff
@@ -22,6 +22,14 @@ function resolveTitle(reportTitle) {
   }
 }
 
+function resolveDefaultSizes(defaultSizes) {
+  return defaultSizes === 'compressed' ? 'gzip' : defaultSizes;
```

Comment:
Can we inverse it please? I mean, deprecate `gzip` default size and use `compressed` instead to it will look like this:
```suggestion
  return defaultSizes === 'gzip' ? 'compressed' : defaultSizes;
```

Reply:
My current approach, based on https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/432#discussion_r613798534, is:
```js
function resolveDefaultSizes(defaultSizes, compressionAlgorithm) {
  if (['gzip', 'brotli'].includes(defaultSizes)) return compressionAlgorithm;
  return defaultSizes;
}
```

That is, we drop `compressed`. The config is "forgiving" if the user specifies the wrong `defaultSizes`.
WDYT?